addons/ballistics/CfgMagazines.hpp

```diff
@@ -8,6 +8,9 @@ class CfgMagazines {
     class 100Rnd_580x42_Mag_F: 30Rnd_580x42_Mag_F {
         initSpeed = 930;
     };
+    class 20Rnd_650x39_Cased_Mag_F: CA_Magazine {
+        initSpeed = 780;
```

Comment:
I set it to 780, since it's the 6.5 cased ammo, which seemed to have higher velicities than the caseless ammo in the vanilla game. Not sure if this makes a lot of sense to do in Arma, but based on how caseless ammo works in the real world, cased ammo probably retains more of the energy as it's a lot harder for the gasses to escape from the action.